### PR TITLE
[SWIEZAKI-8] Database openshift configuration

### PR DESCRIPTION
**Pull request by mateuszwlosek**
Added basic configuration that will allow us to use database in backend app. I check if it works with some examples, but I removed them, we don't want unnecessary code in our repository. I also used .gitignore to clean unnecessary files.

### CODE REVIEW

**Review by MonikaOlkowska — APPROVED** [review]

**Comment by mateuszwlosek** [comment]
Please, do not merge this change. I need to ask some question to person in ADB about the connection between two pods.

File: ConfigurationHandler.cs
Comment (JohnDaarion) [inline ConfigurationHandler.cs:21]: Jeśli dobrze pamiętam to odwołanie się do value w przypadku nulla w konfiguracji sypnie, sprawdź to, niżej to samo

File: DatabaseConnectionProvider.csproj
Comment (JohnDaarion) [inline DatabaseConnectionProvider.csproj:3]: pytanie na boku bawimy się w wersjonowanie?

**Comment by mateuszwlosek** [comment]
Please write any comments in English. I want to gain knowledge from that project and English is the knowledge we all should be seeking.

File: ConfigurationHandler.cs
Comment (mateuszwlosek) [reply ConfigurationHandler.cs:21]: No, I checked. I think that this works similar to gradle configuration. If there is no given key, it will be just an empty string.

File: DatabaseConnectionProvider.csproj
Comment (mateuszwlosek) [reply DatabaseConnectionProvider.csproj:3]: You mean versioning of our whole backend app? Well we can, but versions will be modified while develop will be merged into a master branch. It's not up to me to specify or change project version now. We should discuss that on next meeting, and if we agree to that. Version can be specify herel like I said, before master branch merge.

**Comment by mateuszwlosek** [comment]
My question is: Is a DatabaseConnectionProvider a good name for our backend application?

File: ConfigurationHandler.cs
Comment (JohnDaarion) [reply ConfigurationHandler.cs:21]: What will happen if _configuration.GetSection(someKey) == null and .Value will be requested?

**Comment by JohnDaarion** [comment]
No worries there will be no further comments in polish by me : D
"DatabaseConnectionProvider" is nice.

File: ConfigurationHandler.cs
Comment (mateuszwlosek) [reply ConfigurationHandler.cs:21]: It will return empty String. I checked twice.

**Comment by mateuszwlosek** [comment]
Back end app is not a database connection provider. As a front end app or android app you dont connect to the database by the back end. It does not provide a connection. It provides data, mostly (or maybe all) from the database, but backend app would control whole process of data distribution. Its like a network card. Network card it is not a network provider, of course it allows you to use the network but it does not provide it, it's more like a gateway. I hope that I described the context correctly.

**Pull request merged**
